recipes/mrcz/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "mrcz" %}
+{% set version = "0.3.6" %}
+{% set sha256 = "8b97aa55d2827d0d8371f5a713d6c26a68952c97631a2243b698e0fcb4998992" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+    - setuptools
+    - numpy
+    - futures  # [py2k]
```

Comment:
In theory, we wouldn't have `numpy` or `futures` here, but that depends on changes to the code ( https://github.com/em-MRCZ/python-mrcz/issues/11 ). For now, they are fine to have here until that changes.